src/WebJobs.Script/Description/DotNet/DotNetFunctionInvoker.cs

```diff
@@ -117,6 +113,15 @@ protected override void OnScriptFileChanged(FileSystemEventArgs e)
             }
         }
 
+        private void Shutdown()
```

Comment:
Yeah, I considered that. In this scope, it will only be called in those scenarios, but it would make sense to have a more specific error message. Was trying to avoid additional state and/or additional debounced actions. There's another approach I'll play with to clean this up.